modular_skyrat/modules/modular_items/lewd_items/credits

```diff
@@ -6,6 +6,7 @@ Geminee - Author of many complex and hard features like "milking machine" and ne
 The_CatFish - Helped with many sprites, author of slots icons.
 CoffeeKat - The wizard of gitfuckery and just a good person.
 Useroth - Helped me to learn BYOND coding and made this update possible.
+WeirdButton - For helping with most shibari sprites
```

Comment:
This is nice and all, but this file should be removed, we don't need it, and we specifically don't need it on everyone's repository. You can leave credits in the PR body if you want, but it does not belong on the repo. Nobody else does it, I don't see why you folks would be an exception.